hello-world-python/README.md

```diff
@@ -0,0 +1,110 @@
+# Conclave Hello World Python Sample
+
+This sample shows how to execute a python code inside a conclave enclave.
+Please note this sample uses the hello world sample which is available as part of the conclave sdk.
+Below ``Configurations to be added to a hello world conclave sample`` section talks about the configurations to be added 
+to this sample to enable it to execute python code.
+Conclave support for Python is provided by the polyglot capabilities in GraalVM. 
+Refer to the GraalVM [documentation](https://www.graalvm.org/reference-manual/embed-languages/) on embedding languages 
+for detailed instructions on how to use this capability.
+
+When you use the graalvm-native-image runtime you have the option to enable Python language support. 
+When enabled you can create a context in which you can load and execute a Python source module.
+
+# What are the different modes in which you can run a typical Conclave application
+
+There are four modes in which an enclave can be loaded
+1. Release Mode - You need a real Intel SGX CPU. You load the enclave in release mode when you are in production.
+2. Debug Mode - You need a real Intel SGX CPU. This provides you with certain capabilities whcih enables you to debug
+   your enclave application.
+3. Simulation Mode - This does not require you to have an Intel SGX CPU. This can be used to test your conclave 
+   applications when in development mode. This mode requires a linux env. This can be run inside a Docker container 
+   on Mac. Enclave code is compiled to native binary and loaded inside a separate JVM inside your host JVM.
```

Comment:
Would it be better to say "This mode requires a Linux environment but you can use a Docker container when on a non-Linux machine, such as Windows or MacOS." instead of "This mode requires a linux env. This can be run inside a Docker container on Mac."

Reply:
This work is probably out of scope but people probably don't know how to use a Docker container or which tools they should add to the Docker container image to compile the Conclave sample.

Reply:
I have not added the details on how to run this sample in simulation as as of now we are supporting mock mode for this sample to run. Once we are able to run the sample in simulation mode I will add detailed steps on how to run the sample in simulation mode. What say? Or you want me to give a reference of docs still?

Reply:
It would be better to reference the docs. Conclave SDK contains a bug that needs to be sorted out to allow the enclave to run Python in simulation mode. But that is an issue in Conclave SDK and we should still make an effort to make the sample as complete as possible.

Reply:
Yes ok I agree, I have added the docs link which shows how to run in simulation mode here.